### PR TITLE
[Doc] Make documentation easier to find from repo's root Readme

### PR DESCRIPTION
**Pull request by slax57**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
 An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Auth0](https://auth0.com) instance.
 
+[![Documentation]][DocumentationLink] 
+[![Source Code]][SourceCodeLink] 
+
+[Documentation]: https://img.shields.io/badge/Documentation-green?style=for-the-badge
+[Source Code]: https://img.shields.io/badge/Source_Code-blue?style=for-the-badge
+
+[DocumentationLink]: ./packages/ra-auth-auth0/Readme.md 'Documentation'
+[SourceCodeLink]: https://github.com/marmelab/ra-auth-auth0/tree/main/src/packages/ra-auth-auth0 'Source Code'
```
Comment (slax57) [inline README.md]:
```suggestion
[SourceCodeLink]: https://github.com/marmelab/ra-auth-auth0/tree/main/packages/ra-auth-auth0 'Source Code'
```

**Review by adguernier — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
 
 An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Auth0](https://auth0.com) instance.
 
+[![Documentation]][DocumentationLink] 
+[![Source Code]][SourceCodeLink] 
+
+[Documentation]: https://img.shields.io/badge/Documentation-green?style=for-the-badge
+[Source Code]: https://img.shields.io/badge/Source_Code-blue?style=for-the-badge
+
+[DocumentationLink]: ./packages/ra-auth-auth0/Readme.md 'Documentation'
```
Comment (djhi) [inline README.md:11]: Shouldn't we put an absolute link here for NPM?

Comment (slax57) [reply README.md:11]: We could, but what would be the benefit of that?

Comment (slax57) [reply README.md:11]: Besides, Readme file in GH is always up to date, as opposed to NPM where it's only updated when we make a release. Isn't it better to link to the latest doc version?

**Pull request merged**